app/views/initiatives/index.html.erb

```diff
@@ -25,5 +29,5 @@
         </div>
       <% end %>
     <% end %>
-  </div>
-</div>
+  
+</div>
```

Comment:
I think you may have lost your closing div tag for your `<div class="init-list">` on line 8 here.

Reply:
I added a closing div tag